wordle-web/src/App.vue

```diff
@@ -19,19 +19,26 @@
             <v-btn icon="mdi-hamburger" v-bind="props"></v-btn>
           </template>
 
-          <v-list>
+          <v-list width="200">
             <v-list-item>
               <v-list-item-title>
-                <RouterLink to="/">Play Again</RouterLink>
+                <RouterLink :to="{ name: 'wordOfTheDay' }"> Play Word of the Day </RouterLink>
```

Comment:
It's definitely not required, but it could be nice to have a button in the app bar that displays the last ten days without needing to play a full game to see it.

Reply:
Really good idea, but I struggled to implement it when I tried and ran out of time to keep trying.